examples/v0/account_management/get_accessible_customers.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example lists the resource names for the customer accounts that the
+authenticating user has access to.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+
+import google.ads.google_ads.client
+
+def main(client):
+    customer_service = client.get_service('CustomerService')
+
+    try:
+        accessible_customers = customer_service.list_accessible_customers()
+    except google.ads.google_ads.errors.GoogleAdsException as ex:
+        print('Request with ID "%s" failed with status "%s" and includes the '
+              'following errors:' % (ex.request_id, ex.error.code().name))
+        for error in ex.failure.errors:
+            print('\tError with message "%s".' % error.message)
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print('\t\tOn field: %s' % field_path_element.field_name)
+        sys.exit(1)
+
+    resource_names = accessible_customers.resource_names
+
+    for resource_name in resource_names:
```

Comment:
no need for a one-off throw away name, ie. `for resource_name in accessible_customers.resources_names:`

Reply:
Great, I did keep this in order to slice the result to 200 entries, per your other comment.

Reply:
:+1: